jwst/regtest/test_miri_lrs_slitless.py

```diff
@@ -57,20 +54,18 @@ def generate_tso3_asn():
     """Generate an association file that references the output of run_spec2_pipeline."""
     asn = asn_from_list([f"{DATASET_ID}_calints.fits"], product_name=PRODUCT_NAME)
     asn.data["program"] = "80600"
```

Comment:
I realize this isn't part of this PR, but I'd recommend putting this at the top as a constant -- down here, it's easy for it to fall out of sync with the chosen dataset (as it seems to have done already).